timelock-agent/src/main/java/com/palantir/timelock/paxos/TimelockPaxosAcceptorAdapter.java

```diff
@@ -63,9 +65,23 @@ public long getLatestSequencePreparedOrAccepted() {
      */
     public static Function<Client, List<PaxosAcceptor>> wrap(
             PaxosUseCase paxosUseCase,
-            List<TimelockPaxosAcceptorRpcClient> acceptors) {
-        return client -> acceptors.stream()
-                .map(acceptor -> new TimelockPaxosAcceptorAdapter(paxosUseCase, client, acceptor))
-                .collect(Collectors.toList());
+            PaxosRemoteClients remoteClients) {
+        switch (paxosUseCase) {
+            case LEADER_FOR_ALL_CLIENTS:
+                return _client -> remoteClients.singleLeaderAcceptor().stream()
```

Comment:
is underscore now a fancy way to denote it's unused? I prever just using `unused`

Reply:
baseline will convert everyone to it anyway, I prefer $ but alas, I'm poor